### PR TITLE
zkEVM - Quality assurance 2nd

### PR DESCRIPTION
**Pull request by EmpieichO**
zkEVM - Quality assurance 2nd. Error fixing ... as per issue https://polygon.atlassian.net/browse/DOCS-306

### CODE REVIEW

File: docs/zkEVM/architecture/zkprover/index.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ The zkProver mainly interacts with two components, i.e. the Node and the databas
 
 As depicted in the flow diagram above, the whole interaction works out in 4 steps.
 
-1. The Node sends the content of Merkle trees to the database to be stored there
+1. The Node sends the content of Merkle trees to the database to be stored there.
```
Comment (hsutaiyu) [inline docs/zkEVM/architecture/zkprover/index.md]:
```suggestion
1. The node sends the content of Merkle trees to the database to be stored there.
```

File: docs/zkEVM/architecture/zkprover/index.md [hunk 2]
```diff
@@ -15,13 +15,13 @@ The zkProver mainly interacts with two components, i.e. the Node and the databas
...
 
-2. The Node then sends the input transactions to the zkProver
+2. The Node then sends the input transactions to the zkProver.
```
Comment (hsutaiyu) [inline docs/zkEVM/architecture/zkprover/index.md]:
```suggestion
2. The node then sends the input transactions to the zkProver.
```

File: docs/zkEVM/architecture/zkprover/index.md [hunk 3]
```diff
@@ -158,21 +158,21 @@ The Executor executes all instructions on top of the PIL hardware and generates
 
 Once the Main state machine Executor has converted transactions and related data to committed polynomials, the STARK Recursion Component takes the following inputs;
 
-1. The Committed Polynomials,
-2. The Constant Polynomials,
-3. Scripts, which are lists of instructions,
+1. The Committed Polynomials.
+2. The Constant Polynomials.
```
Comment (hsutaiyu) [inline docs/zkEVM/architecture/zkprover/index.md]: Please confirm the capitalization of C here.

**Review by hsutaiyu — APPROVED** [review]

File: docs/zkEVM/architecture/zkprover/index.md (hunk 3 above)
Comment (EmpieichO) [reply docs/zkEVM/architecture/zkprover/index.md]: Fixing that. Thanks

**Pull request merged**